### PR TITLE
added rules comments for first and second 21 stubs

### PR DESCRIPTION
**Pull request by brianpmaher**
This PR covers Tickets 42 and 43.

bandaged-hands
- Added comments for rules for the first 42 stubs
- Updated one inconsistency I found. Could be more, just noticed that one.

### CODE REVIEW

File: parser.cs [hunk 1]
```diff
@@ -422,7 +465,9 @@ public class Parser {
 
     private void emptyStatement(){
         switch(__lookahead.Type) {
+            case TOKENS.END:
```
Comment (JesseStephenBrown) [inline parser.cs:468]: Nice catch. Did not review these hard.

**Comment by JesseStephenBrown** [comment]
+1

File: parser.cs (hunk 1 above)
Comment (brianpmaher) [inline parser.cs:468]: Not sure if there's more, but this was the only one I found. It's likely there's more but they will stand out during parsing.

**Comment by seanrogers2657** [comment]
+1 Looks good!

**Pull request merged**